retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss

```diff
@@ -1027,20 +1027,26 @@ QPushButton {
   padding: 2px;
   outline: none;
   border: none;
+  padding-left: 6px;
+  padding-right: 6px;
```

Comment:
Can you move this in our area after this line https://github.com/RetroShare/RetroShare/blob/master/retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss#L2229 ?
Prefer don't change the QDarkStyleSheet part, this will be more easy to update it.
Look how I did for QTreeView::item
You only need to rewrite what you have to change to overide what QDarkStyleSheet does.